content/en/language_clients/javascript/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+type: docs
+category: JavaScript
+title: JavaScript Client Overview
+weight: 5
+---
+
+[sigstore-js](https://github.com/sigstore/sigstore-js) is a collection of [javascript libraries](#additional-packages) for interacting with Sigstore.
+
+The main package,[`sigstore`](https://www.npmjs.com/package/sigstore), is a JavaScript library for generating and verifying Sigstore signatures. One of the intended uses is to sign and verify npm packages but it can be used to sign and verify any file.
+
+Full project documentation can be found in the [sigstore-js](https://github.com/sigstore/sigstore-js#sigstore-js---) project README and in each [package README](https://github.com/sigstore/sigstore-js/tree/main/packages).
+
+## Features
+
+- Support for signing using an OpenID Connect identity
+- Support for publishing signatures to a [Rekor](../../logging/overview/) instance
+- Support for verifying Sigstore bundles
+
+## Installation
+
+[`sigstore`](https://www.npmjs.com/package/sigstore) requires Node.js version >= 18.17.0.
+
+To install `sigstore` run the following command:
+
+```console
+npm install sigstore
+```
+
+## Example
+
+ To use `sigstore`, import the following into your project using the following:
+
+ ```console
+ const { sign, verify } = require('sigstore');
+ ```
+
+ ```console
+ import { sign, verify } from 'sigstore';
+ ```
+
+### Signing example
+
+ The following function will sign the file `foo.txt`:
+
+ ```console
+ sign('foo.txt');
+ ```
+
+ There are a number of optional arguments that can be used with the sign function. Additional information is available in the [project repository](https://github.com/sigstore/sigstore-js/tree/main/packages/client#signpayload-options).
+
+ The sign function will return a Sigstore bundle (`foo.txt.sigstore.json`) which includes the signature and the necessary verification material.
```

Comment:
The returned value will be a JSON-encoded bundle, not a file name -- you might want to ditch the reference to `foo.txt.sigstore.json` here.